src/ipc/ipc4/handler.c

```diff
@@ -65,7 +65,7 @@ struct ipc4_msg_data {
 static struct ipc4_msg_data msg_data;
 
 /* fw sends a fw ipc message to send the status of the last host ipc message */
-static struct ipc_msg msg_reply;
+static struct ipc_msg msg_reply = {0, 0, 0, 0, {&msg_reply.list, &msg_reply.list}};
```

Comment:
maybe:
static struct ipc_msg msg_reply = {0, 0, 0, 0, LIST_INIT(&msg_reply.list)};

Reply:
thanks, refined,  it should be static struct ipc_msg msg_reply = {0, 0, 0, 0, LIST_INIT(msg_reply.list)};